### PR TITLE
CNNETANTWO-52 and All subtasks

### PR DESCRIPTION
**Pull request by zoitec**
CNNETANTWO-53 - content list page
CNNETANTWO-54 - content list module
CNNETANTWO-55 - content module unit tests

Pull this branch down and go to http://localhost:5050/list

image

### CODE REVIEW

**Comment by jamsyoung** [comment]
This is what I get on my localhost when running `npm run start-docker-local`

**Comment by dsover** [comment]
This is an issue with the package-lock.json file. I think it is trying to keep old dependancies of the cnn-starter and because it is an internal (git) dependency it is getting confused. I was able to remove the file, run npm i it worked. Then I ran `npm run start-docker-local` and everything was working correctly.

File: src/redux/actions/articleActions.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import * as types from './actionTypes'
```
Comment (dsover) [inline src/redux/actions/articleActions.js]: Name this articlesActions to match the reducer name

File: src/modules/article-item/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import get from 'lodash/get';
+
+import { Card, CardText, CardMedia } from 'react-toolbox/lib/card';
+
+
+const propTypes = {
+    articleDocs: PropTypes.array
+};
+
+const ArticleItem = ({ articleDocs }) => {
```
Comment (dsover) [inline src/modules/article-item/index.js]: This should be called ArticleItems as it's an array

**Pull request closed without merging**